scripts/install_poetry.py

```diff
@@ -16,12 +16,8 @@
     * Attempt to inform the user if they need to add this bin directory to their `$PATH`, as well as how to do so.
     * Upon failure, write an error log to `poetry-installer-error-<hash>.log and restore any previous environment.
 
-This script performs minimal magic, and should be relatively stable. However, it is optimized for interactive developer
-use and trivial pipelines. If you are considering using this script in production, you should consider manually-managed
-installs, or use of pipx as alternatives to executing arbitrary, unversioned code from the internet. If you prefer this
-script to alternatives, consider maintaining a local copy as part of your infrastructure.
```

Comment:
As we can see from the original, poetry themselves recommends maintaining our own copy over installing one from the internet